web/ViewModels/LandlordProfileModel.cs

```diff
@@ -68,4 +68,23 @@ public static LandlordProfileModel FromDbModel(LandlordDbModel landlord, BricksA
             CurrentUserIsAdmin = user.IsAdmin
         };
     }
+
+    public static LandlordProfileModel FromDbModel(LandlordDbModel landlord)
+    {
+        return new LandlordProfileModel
+        {
+            LandlordId = landlord.Id,
+            CompanyName = landlord.CompanyName,
+            Email = landlord.Email,
+            FirstName = landlord.FirstName,
+            LastName = landlord.LastName,
+            Phone = landlord.Phone,
+            Title = landlord.Title,
+            LandlordType = landlord.LandlordType,
+            LandlordProvidedCharterStatus = landlord.LandlordProvidedCharterStatus,
+            CharterApproved = landlord.CharterApproved,
+            IsLandlordForProfit = landlord.IsLandlordForProfit,
+            CurrentUserIsAdmin = landlord.User?.IsAdmin ?? false

```

Comment:
We should probably remove `CurrentUserIsAdmin` from LandlordProfileModel and either use `@User.HasRole` or have a separate view model for the page(s) that require it.

If you've got time to do that would you mind making a separate PR for it? If not no worries.

Reply:
I'll do a separate PR for this while Blanca is working on our editing property bits :)